.travis.yml

```diff
@@ -21,6 +21,21 @@ after_success:
 
 jobs:
   include:
+    - stage: loopback.io
+      before_install:
+        - |
+          if git diff --name-only --quiet origin/master docs/; then
```

Comment:
Can we use `$TRAVIS_BRANCH` instead of `origin/master`? I am worried that a hard-coded branch name `origin/master` will not work in the future, once we create branches like `v4.x`.

See https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

Reply:
I think it might be even better to use `$TRAVIS_COMMIT_RANGE`